### ISSUE TITLE
Have a breeding map alongside normal seasonal / detection maps

### ISSUE THREAD

**Issue by Louis-Backstrom** [bird-team/brisbane-bird-atlas#123]
I think it would be helpful (especially when writing accounts, but also more generally) to have another map type - showing all cells where breeding has been reported for the species. This could be just a straight binary map (ie red for yes, blank for no breeding data for the square) or it could be based on % of reporting checklists that have breeding codes added.

Once again, low priority so I'll hibernate it for now.

**Comment by jeffreyhanson** [comment]
Yeah, I like this idea - but I wonder if most species have enough breeding data to provide an informative map (rather than just a few red grid cells here and there)? Also, if we add this map then we'll have 7 maps, which we can't arrange nicely on a grid. So we might also want to think of another map/graph so we can have a the maps/graphs laid out in a 2x4 grid?

**Comment by Louis-Backstrom** [comment]
I agree that we won't have much data initially, but I think the ideal is that we get more over the course of the project and eventually most common breeders have a decently-filled in map.

As for arranging - could we do one "breeding detection" (binary yes/no) and one "breeding frequency" (scale 0-100%) similar to the other maps?
